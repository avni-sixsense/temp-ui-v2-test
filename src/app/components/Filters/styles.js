import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  inputBase: {
    paddingTop: '8px',
    '& .MuiAutocomplete-inputRoot': {
      width: '90%',
      margin: 'auto'
    },
    '& .MuiAutocomplete-endAdornment': {
      display: 'none'
    },
    '& .MuiInput-underline:after': {
      borderBottom: '1px solid #E8EDF1'
    },
    '& .MuiInput-underline:before': {
      borderBottom: '1px solid #E8EDF1'
    }
  },
  paper: {
    width: '300px',
    maxWidth: '300px',
    boxShadow: 'none',
    margin: 0,
    maxHeight: 270
  },
  paperDefect: {
    boxShadow: 'none',
    margin: 0
    // maxHeight: 396
  },
  option: {
    minHeight: 'auto',
    alignItems: 'flex-start',
    paddingTop: 0,
    paddingBottom: 16,
    paddingLeft: 15,
    paddingRight: 8,

    '&[aria-selected="true"]': {
      backgroundColor: 'transparent'
    },
    '&[data-focus="true"]': {
      backgroundColor: 'transparent'
    },
    '& .MuiAutocomplete-option': {
      paddingLeft: '0px'
    }
  },
  popperDisablePortal: {
    position: 'relative'
  },
  popperDisablePortalDefect: {
    position: 'relative',
    width: '300px !important'
  },
  item: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer'
  },
  popper: {
    zIndex: 100
  },
  menu: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '300px',
      maxHeight: '356px',
      '& .MuiAutocomplete-listbox': {
        margin: '12px 0',
        padding: 0
      },
      '& .MuiMenu-list': {
        paddingLeft: '0 !important',
        paddingRight: '0 !important',
        paddingTop: '0 !important',
        paddingBottom: '0 !important',
        height: '100%'
      }
    }
  },
  menuDefect: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '300px',
      height: '402px',
      '& .MuiMenu-list': {
        paddingLeft: '0 !important',
        paddingRight: '0 !important',
        paddingTop: '0 !important',
        paddingBottom: '0 !important',
        height: '100%'
      },
      '& .MuiAutocomplete-option': {
        padding: 0
      }
    }
  },
  dateMenu: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '340px',
      minHeight: '356px',
      padding: '8px',
      '& .MuiMenu-list': {
        padding: '0 !important',
        width: '100% !important',
        height: '100%'
      }
    }
  },
  menuItem: {
    // position: 'absolute',
    // bottom: 0,
    backgroundColor: 'transparent',
    '&:hover': {
      backgroundColor: 'transparent'
    },
    '&:focus': {
      backgroundColor: 'transparent'
    }
  },
  list: {
    height: '250px',
    overflowY: 'scroll'
  },
  search: {
    width: '80%',
    border: 'none',
    borderBottom: '0.5px solid #E3E3E3',
    fontSize: '12px',
    lineHeight: '32px',
    color: '#000000',
    opacity: 0.5
  },
  cancelBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#02435D',
    backgroundColor: 'rgba(2, 67, 93, 0.06)',
    padding: '8px 24px',
    '&:hover': {
      backgroundColor: 'rgba(2, 67, 93, 0.12)'
    },
    '&:disabled': {
      background: '#E0E0E0',
      color: '#000D12',
      opacity: 0.24
    }
  },
  applyBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#FFFFFF',
    backgroundColor: '#F56C6C !important',
    padding: theme.spacing(1, 3),
    '&:hover': {
      backgroundColor: '#E14242 !important'
    },
    '&:disabled': {
      background: '#E0E0E0 !important',
      color: '#000D12',
      opacity: 0.24
    }
  },
  datePicker: {
    backgroundColor: 'yellow'
  }
}));

export default useStyles;
