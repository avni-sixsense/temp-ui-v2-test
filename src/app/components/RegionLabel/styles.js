// @flow

// import { grey } from '@material-ui/core/colors'

export default {
  paper: {
    boxShadow: 'none',
    backgroundColor: '#1C2A42',
    color: '#FFFFFF',
    '& p': {
      color: '#FFFFFF'
    },
    '& .MuiAutocomplete-listbox': {
      '& .MuiListSubheader-gutters': {
        fontSize: '0.875rem',
        color: '#FFFFFF',
        backgroundColor: '#1C2A42'
      }
      // '& .MuiAutocomplete-groupUl .MuiAutocomplete-option': {
      // 	display: 'flex',
      // 	alignItems: 'center',
      // 	fontWeight: 400,
      // },
    }
  },
  option: {
    // minHeight: 'auto',
    // alignItems: 'flex-start',
    // padding: 0,
    '&[aria-selected="true"]': {
      backgroundColor: 'transparent'
    },
    '&[data-focus="true"]': {
      backgroundColor: 'transparent'
    }
    // backgroundColor: '#1C2A42',
    // color: '#FFFFFF',
  },
  popperDisablePortal: {
    position: 'relative',
    width: 'auto !important',
    maxHeight: 184,
    overflowY: 'auto',
    '&::-webkit-scrollbar ': {
      width: 3
    },

    /* Track */
    '&::-webkit-scrollbar-track': {
      borderRadius: 10
    },

    /* Handle */
    '&::-webkit-scrollbar-thumb': {
      background: '#31456A',
      borderRadius: 10
    },

    /* Handle on hover */
    '&::-webkit-scrollbar-thumb:hover': {
      background: '#EEEEEE'
    }
  },
  item: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer'
  },
  popper: {
    zIndex: 100
  },
  label: {
    fontSize: '0.875rem',
    fontWeight: 400,
    color: '#313131'
  },
  button: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#FFFFFF',
    backgroundColor: '#F56C6C',
    padding: '8px 24px',
    '&:hover': {
      backgroundColor: '#E14242'
    },
    '&:disabled': {
      background: '#E0E0E0',
      color: '#000D12',
      opacity: 0.24
    }
  },
  regionInfo: {
    // cursor: 'default',
    // transition: 'opacity 200ms',
    // '&:hover': {
    // 	cursor: 'pointer',
    // },
    // pointerEvents: "none",
    padding: 6,
    '& .name': {
      display: 'flex',
      flexDirection: 'row',
      alignItems: 'center',
      '& .circle': {
        marginRight: 4,
        boxShadow: '0px 0px 2px rgba(0,0,0,0.4)',
        width: 10,
        height: 10,
        borderRadius: 5
      }
    },
    '& .tags': {
      display: 'flex',
      '& .tag': {
        // color: grey[700],
        // display: 'inline-block',
        // margin: 1,
        // fontSize: 10,
        // textDecoration: 'underline',
        backgroundColor: 'rgba(37, 99, 235, 0.8)',
        color: '#FFFFFF',
        borderRadius: '2px',
        display: 'flex',
        alignItems: 'center'
      },
      '& .addIcon': {
        cursor: 'pointer'
      }
    }
  },
  open: {
    minWidth: '175px',
    borderRadius: 0,
    '& .MuiAutocomplete-endAdornment': {
      display: 'none'
    }
  },
  closeBtn: {
    fontSize: '12px',
    cursor: 'pointer'
  },
  inputBase: {
    borderRadius: '8px',
    width: '235px !important',
    backgroundColor: '#162236',
    color: '#FFFFFF',
    '& input': {
      color: '#FFFFFF',
      '&::placeholder': {
        color: '#31456A',
        opacity: 1
      },
      '&:-ms-input-placeholder': {
        color: '#31456A'
      },
      '&::-ms-input-placeholder': {
        color: '#31456A'
      }
    },
    '& .MuiAutocomplete-tag': {
      backgroundColor: '#7C3AED',
      color: '#FFFFFF',
      fontSize: '0.8125rem'
    },
    '& .MuiOutlinedInput-notchedOutline': {
      // border: '1px solid #4F46E5',
      // boxShadow: '0px 0px 5px #C7D2FE',
      // '&:hover': {
      // 	border: '1px solid #4F46E5',
      // 	boxShadow: '0px 0px 5px #C7D2FE',
      // },
      borderRadius: '8px'
    }
  },
  searchIcons: {
    paddingTop: '0px',
    paddingBottom: '0px',
    paddingRight: '0px'
  },
  tags: {
    borderRadius: '4px',
    color: '#FFFFFF',
    '& .MuiTypography-body1': {
      fontSize: '0.8125rem !important'
    },
    '& button': {
      padding: 0,
      color: '#FFFFFF',
      opacity: 0.5
    },
    '& .MuiSvgIcon-root': {
      fontSize: '0.8125rem !important'
    }
  }
};
