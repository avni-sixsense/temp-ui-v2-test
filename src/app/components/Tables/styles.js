import { makeStyles } from '@material-ui/core/styles';
import { alpha } from '@material-ui/core/styles/colorManipulator';

const useStyles = makeStyles(theme => ({
  tableContainer: {
    marginTop: theme.spacing(1),
    position: 'relative',
    minHeight: '125px',
    overflowY: 'hidden'
  },
  loadingIndicator: {
    padding: theme.spacing(2, 0)
  },
  firstCell: {
    padding: theme.spacing(0, 0, 0, 3),
    paddingTop: 0
  },
  divider: {
    width: '100%',
    height: '48px',
    display: 'flex',
    justifyContent: 'center',
    flexDirection: 'column',
    alignItems: 'center'
  },
  dot: {
    width: '6px',
    height: '6px',
    marginLeft: '0.5px',
    background: '#B3C7CE',
    borderRadius: '5px'
  },
  dash: {
    height: '20px',
    borderLeft: '0.5px dashed #000000'
  },
  expand: {
    borderTop: ' 2px solid #E6E6E6',
    padding: theme.spacing(2, 0)
  },
  records: {
    background: 'rgba(223,239,245,0.5)'
  },
  items: {
    borderTop: '2px solid #E6E6E6'
  },
  img: {
    width: '100px',
    height: '100px',
    marginRight: '8px',
    borderRadius: 8,
    marginBottom: 8
  },
  cell: {
    display: 'flex !important',
    paddingLeft: '0px !important'
  },
  headCell: {
    borderBottom: '2px solid rgba(230, 230, 230, 0.5)',
    paddingTop: 0
  },
  text: {
    zIndex: 100,
    marginTop: '-65px',
    marginLeft: '22px',
    color: 'white'
  },
  overlayWrapper: {
    position: 'absolute',
    top: 0,
    left: 0,
    height: '100%',
    width: '100%',
    zIndex: 11
  },
  overlay: {
    display: 'table',
    width: '100%',
    height: '100%',
    backgroundColor: alpha(theme.palette.background.paper, 0.7)
  },
  overlayInner: {
    display: 'table-cell',
    width: '100%',
    height: '100%',
    verticalAlign: 'middle',
    textAlign: 'center'
  },
  subRow: {
    '& .MuiTableCell-root': {
      paddingTop: 0,
      paddingBottom: 0,
      verticalAlign: 'sub'
    }
  },
  tableBody: {
    '& .MuiTableRow-root': {
      '& .MuiTableCell-root': {
        border: 'none',
        wordBreak: 'break-word'
      }
    }
  }
}));
export default useStyles;
