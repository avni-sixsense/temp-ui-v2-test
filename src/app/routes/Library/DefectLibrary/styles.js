import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  paperBg: {
    backgroundColor: '#FFFFFF'
  },
  root: {
    width: '100%'
  },
  paper: {
    width: '100%',
    marginBottom: theme.spacing(2)
  },
  table: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px !important'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  tableHeadBorder: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px',
      borderBottom: 'none'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  visuallyHidden: {
    border: 0,
    clip: 'rect(0 0 0 0)',
    height: 1,
    margin: -1,
    overflow: 'hidden',
    padding: 0,
    position: 'absolute',
    top: 20,
    width: 1
  },
  divider: {
    width: '100%',
    height: '48px',
    display: 'flex',
    justifyContent: 'center',
    flexDirection: 'column',
    alignItems: 'center'
  },
  dot: {
    width: '6px',
    height: '6px',
    marginLeft: '0.5px',
    background: '#B3C7CE',
    borderRadius: '5px'
  },
  dash: {
    height: '20px',
    borderLeft: '0.5px dashed #000000'
  },
  solid: {
    height: '20px',
    borderLeft: 'none'
  },
  tableSection: {
    marginTop: theme.spacing(4)
  }
}));

export default useStyles;
