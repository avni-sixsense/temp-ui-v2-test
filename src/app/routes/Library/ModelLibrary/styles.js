import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  paperBg: {
    backgroundColor: '#FFFFFF'
  },
  root: {
    width: '100%'
  },
  paper: {
    width: '100%',
    marginBottom: theme.spacing(2)
  },
  tableSection: {
    marginTop: theme.spacing(4)
  },
  table: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  visuallyHidden: {
    border: 0,
    clip: 'rect(0 0 0 0)',
    height: 1,
    margin: -1,
    overflow: 'hidden',
    padding: 0,
    position: 'absolute',
    top: 20,
    width: 1
  },
  divider: {
    width: '100%',
    height: '48px',
    display: 'flex',
    justifyContent: 'center',
    flexDirection: 'column',
    alignItems: 'center'
  },
  dot: {
    width: '6px',
    height: '6px',
    marginLeft: '0.5px',
    background: '#B3C7CE',
    borderRadius: '5px'
  },
  dash: {
    height: '20px',
    borderLeft: '0.5px dashed #000000'
  },
  solid: {
    height: '20px',
    borderLeft: 'none'
  },
  tableContainer: {
    overflow: 'hidden',
    maxHeight: 'calc(100vh - 200px)',
    '& [class*="MuiTableContainer-root"]': {
      overflow: 'auto',
      maxHeight: 'calc(100vh - 360px)',
      // maxHeight: '1500px',
      '&::-webkit-scrollbar': {
        width: '8px'
      },
      '&::-webkit-scrollbar-track': {
        boxShadow: 'inset 0 0 0px white',
        borderRadius: '5px'
      },
      '&::-webkit-scrollbar-thumb': {
        backgroundColor: ' #dfdcdc',
        borderRadius: '10px'
      },
      '&::-webkit-scrollbar-thumb:hover': {
        backgroundColor: '#cecece'
      }
    }
  }
}));

export default useStyles;
