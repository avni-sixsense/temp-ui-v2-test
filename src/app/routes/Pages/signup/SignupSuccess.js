import Grid from '@material-ui/core/Grid';
import Paper from '@material-ui/core/Paper';
import { makeStyles } from '@material-ui/core/styles';
import Typography from '@material-ui/core/Typography';
import Success from 'assests/images/group-29.svg';
import React from 'react';

const useStyles = makeStyles(theme => ({
  paper: {
    width: '413px',
    display: 'flex',
    flexDirection: 'column',
    padding: 0,
    marginTop: theme.spacing(5),
    backgroundColor: 'transparent',
    marginBottom: theme.spacing(2),
    marginLeft: 'auto',
    marginRight: 'auto'
  },
  title: {
    ...theme.typeFace.futura125,
    fontWeight: 500,
    lineHeight: '2rem',
    textAlign: 'center',
    marginTop: theme.spacing(3)
  },
  subTitle: {
    ...theme.typeFace.futuraBT1,
    lineHeight: '1.5rem',
    textAlign: 'center',
    opacity: 0.8,
    marginTop: theme.spacing(0.5),
    marginBottom: theme.spacing(4)
  },
  successImg: {
    margin: 'auto',
    width: '248px',
    height: '182px'
  }
}));

const SignupSuccess = () => {
  const classes = useStyles();
  return (
    <>
      <Paper elevation={0} className={classes.paper}>
        <Grid container className={classes.container} justifyContent='center'>
          <Grid item>
            <img src={Success} className={classes.successImg} alt='' />
          </Grid>
          <Grid item>
            <Typography variant='h1' className={classes.title}>
              Signup Successful
            </Typography>
            <Typography variant='h1' className={classes.subTitle}>
              We are processing your request from our end. Your admin will be
              sending out the credentials over the email.
            </Typography>
          </Grid>
        </Grid>
      </Paper>
    </>
  );
};

export default SignupSuccess;
