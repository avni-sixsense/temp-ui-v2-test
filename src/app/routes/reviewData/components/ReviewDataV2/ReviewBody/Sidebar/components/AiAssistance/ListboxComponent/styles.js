import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  inputBase: {
    borderRadius: '8px',
    width: '100% !important',
    backgroundColor: '#162236',
    color: '#FFFFFF',
    '& input': {
      color: '#FFFFFF',
      '&::placeholder': {
        color: '#31456A',
        opacity: 1
      },
      '&:-ms-input-placeholder': {
        color: '#31456A'
      },
      '&::-ms-input-placeholder': {
        color: '#31456A'
      }
    },
    '& .MuiAutocomplete-tag': {
      backgroundColor: '#7C3AED',
      color: '#FFFFFF',
      fontSize: '0.8125rem'
    },
    '& .MuiOutlinedInput-notchedOutline': {
      borderRadius: '8px'
    }
  },
  lightInputBase: {
    borderRadius: '8px',
    width: '100% !important',
    backgroundColor: theme.colors.grey[2],
    color: theme.colors.grey[21],
    '& input': {
      color: theme.colors.grey[21],
      '&::placeholder': {
        color: theme.colors.grey[10],
        opacity: 1
      },
      '&:-ms-input-placeholder': {
        color: theme.colors.grey[10]
      },
      '&::-ms-input-placeholder': {
        color: theme.colors.grey[10]
      }
    },
    '& .MuiAutocomplete-tag': {
      backgroundColor: theme.colors.grey[0],
      color: theme.colors.grey[19],
      fontSize: '0.8125rem'
    },
    '& .MuiOutlinedInput-notchedOutline': {
      borderRadius: '8px'
    },
    '& :hover': {
      '& [class*="MuiOutlinedInput-notchedOutline-"]': {
        borderColor: theme.colors.blue[600],
        borderWidth: '1px',
        boxShadow: '0px 0px 10px rgba(37, 99, 235, 0.1), 0px 0px 1px #2563EB'
      }
    },
    '& .Mui-focused': {
      '& [class*="MuiOutlinedInput-notchedOutline-"]': {
        borderColor: theme.colors.blue[600],
        borderWidth: '1px',
        boxShadow: '0px 0px 10px rgba(37, 99, 235, 0.1), 0px 0px 1px #2563EB'
      }
    }
  },
  paper: {
    width: '300px',
    wordWrap: 'break-word',
    maxWidth: '300px',
    boxShadow: 'none',
    backgroundColor: '#1C2A42',
    color: '#FFFFFF',
    '& p': {
      color: '#FFFFFF',
      fontSize: '0.875rem !important'
    },
    '& li': {
      '& p': {
        msWordBreak: 'break-all',
        overflowX: 'auto'
      },
      '& div': {
        fontSize: '0.875rem',
        color: '#FFFFFF',
        backgroundColor: '#1C2A42'
      },
      paddingLeft: 0
    },
    '& ul': {
      overflowY: 'scroll',
      // maxHeight: '200px',
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: '#EEEEEE'
      }
    }
    // maxHeight: 190,
    // overflowY: 'scroll',
    // '&::-webkit-scrollbar ': {
    // 	width: 3,
    // },

    // /* Track */
    // '&::-webkit-scrollbar-track': {
    // 	borderRadius: 10,
    // },

    // /* Handle */
    // '&::-webkit-scrollbar-thumb': {
    // 	background: '#31456A',
    // 	borderRadius: 10,
    // },

    // /* Handle on hover */
    // '&::-webkit-scrollbar-thumb:hover': {
    // 	background: '#EEEEEE',
    // },
  },
  lightPaper: {
    width: '300px',
    wordWrap: 'break-word',
    maxWidth: '300px',
    boxShadow: 'none',
    backgroundColor: theme.colors.grey[0],
    color: theme.colors.grey[19],
    '& p': {
      color: theme.colors.grey[19],
      fontSize: '0.875rem !important'
    },
    '& li': {
      '& p': {
        msWordBreak: 'break-all',
        overflowX: 'auto'
      },
      '& div': {
        fontSize: '0.875rem',
        color: theme.colors.grey[19],
        backgroundColor: theme.colors.grey[0]
      },
      paddingLeft: 0
    },
    '& ul': {
      overflowY: 'scroll',
      // maxHeight: '200px',
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: theme.colors.grey[4]
      }
    }
    // maxHeight: 190,
    // overflowY: 'scroll',
    // '&::-webkit-scrollbar ': {
    // 	width: 3,
    // },

    // /* Track */
    // '&::-webkit-scrollbar-track': {
    // 	borderRadius: 10,
    // },

    // /* Handle */
    // '&::-webkit-scrollbar-thumb': {
    // 	background: theme.colors.grey[0],
    // 	borderRadius: 10,
    // },

    // /* Handle on hover */
    // '&::-webkit-scrollbar-thumb:hover': {
    // 	background: theme.colors.grey[4],
    // },
  },
  option: {
    '&[aria-selected="true"]': {
      backgroundColor: theme.colors.grey[17]
    },
    '&[data-focus="true"]': {
      backgroundColor: theme.colors.grey[16]
    },
    '& div': {
      backgroundColor: 'transparent !important'
    }
  },
  lightOption: {
    '&[aria-selected="true"]': {
      backgroundColor: theme.colors.grey[2]
    },
    '&[data-focus="true"]': {
      backgroundColor: theme.colors.grey[1]
    },
    '& div': {
      backgroundColor: 'transparent !important'
    }
  },
  popperDisablePortal: {
    position: 'relative',
    width: 'auto !important',
    // maxHeight: 184,
    overflowY: 'auto',
    backgroundColor: '#1C2A42',
    color: '#FFFFFF',
    '&::-webkit-scrollbar ': {
      width: 3
    },

    /* Track */
    '&::-webkit-scrollbar-track': {
      borderRadius: 10
    },

    /* Handle */
    '&::-webkit-scrollbar-thumb': {
      background: '#31456A',
      borderRadius: 10
    },

    /* Handle on hover */
    '&::-webkit-scrollbar-thumb:hover': {
      background: '#EEEEEE'
    }
  },
  lightPopperDisablePortal: {
    position: 'relative',
    width: 'auto !important',
    // maxHeight: 184,
    overflowY: 'auto',
    backgroundColor: theme.colors.grey[0],
    color: theme.colors.grey[19],
    '&::-webkit-scrollbar ': {
      width: 3
    },

    /* Track */
    '&::-webkit-scrollbar-track': {
      borderRadius: 10
    },

    /* Handle */
    '&::-webkit-scrollbar-thumb': {
      background: theme.colors.grey[0],
      borderRadius: 10
    },

    /* Handle on hover */
    '&::-webkit-scrollbar-thumb:hover': {
      background: theme.colors.grey[4]
    }
  },
  popperDisablePortalDefect: {
    position: 'relative',
    width: '300px !important'
  },
  item: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer',
    backgroundColor: theme.colors.grey[14],
    padding: theme.spacing(0.25, 0.75, 0.5, 0.75),
    borderRadius: '2px',
    boxShadow: theme.colors.shadow['base'],
    '& p': {
      color: theme.colors.grey[0],
      fontWeight: 600,
      fontSize: '0.875rem',
      marginRight: theme.spacing(0.6875)
    },
    '& svg': {
      color: theme.colors.grey[8],
      fontWeight: 900,
      fontSize: '0.625rem'
    },
    '& ul': {
      padding: '0px !important'
    }
  },
  lightItem: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer',
    border: `0.8px solid ${theme.colors.grey[4]}`,
    backgroundColor: theme.colors.grey[0],
    padding: theme.spacing(0.25, 0.75, 0.5, 0.75),
    borderRadius: '2px',
    boxShadow: theme.colors.shadow['base'],
    '& p': {
      color: theme.colors.grey[19],
      fontWeight: 600,
      fontSize: '0.875rem',
      marginRight: theme.spacing(0.6875)
    },
    '& svg': {
      color: theme.colors.grey[7],
      fontWeight: 900,
      fontSize: '0.625rem'
    },
    '& ul': {
      padding: '0px !important'
    }
  },
  popper: {
    zIndex: 100
  },
  menuPaper: {
    backgroundColor: '#1C2A42',
    padding: theme.spacing(1)
  },
  lightMenuPaper: {
    backgroundColor: theme.colors.grey[0],
    padding: theme.spacing(1),
    border: `0.5px solid ${theme.colors.grey[4]}`
  },
  menuDefect: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '300px',
      height: '402px',
      '& .MuiMenu-list': {
        paddingLeft: '0 !important',
        paddingRight: '0 !important',
        paddingTop: '0 !important',
        paddingBottom: '0 !important',
        height: '100%'
      },
      '& .MuiAutocomplete-option': {
        padding: 0
      }
    }
  },
  dateMenu: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '340px',
      minHeight: '356px',
      padding: '8px',
      '& .MuiMenu-list': {
        padding: '0 !important',
        width: '100% !important',
        height: '100%'
      }
    }
  },
  menuItem: {
    position: 'absolute',
    bottom: 0,
    backgroundColor: 'transparent',
    '&:hover': {
      backgroundColor: 'transparent'
    },
    '&:focus': {
      backgroundColor: 'transparent'
    }
  },
  list: {
    height: '250px',
    overflowY: 'scroll'
  },
  search: {
    width: '80%',
    border: 'none',
    borderBottom: '0.5px solid #E3E3E3',
    fontSize: '12px',
    lineHeight: '32px',
    color: '#000000',
    opacity: 0.5
  },
  cancelBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#02435D',
    backgroundColor: 'rgba(2, 67, 93, 0.06)',
    padding: '8px 24px',
    '&:hover': {
      backgroundColor: 'rgba(2, 67, 93, 0.12)'
    },
    '&:disabled': {
      background: '#E0E0E0',
      color: '#000D12',
      opacity: 0.24
    }
  },
  applyBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#FFFFFF',
    backgroundColor: '#F56C6C !important',
    padding: theme.spacing(1, 3),
    '&:hover': {
      backgroundColor: '#E14242 !important'
    },
    '&:disabled': {
      background: '#E0E0E0 !important',
      color: '#000D12',
      opacity: 0.24
    }
  },
  datePicker: {
    backgroundColor: 'yellow'
  },
  filterCount: {
    backgroundColor: theme.colors.grey[10],
    color: theme.colors.grey[0],
    borderRadius: '1000px',
    paddingLeft: theme.spacing(0.5),
    paddingRight: theme.spacing(0.5),
    marginLeft: theme.spacing(0.5),
    marginRight: theme.spacing(0.25),
    paddingBottom: theme.spacing(0.125)
  },
  lightfilterCount: {
    backgroundColor: theme.colors.grey[2],
    color: theme.colors.grey[19],
    borderRadius: '1000px',
    paddingLeft: theme.spacing(0.5),
    paddingRight: theme.spacing(0.5),
    marginLeft: theme.spacing(0.5),
    marginRight: theme.spacing(0.25),
    paddingBottom: theme.spacing(0.125)
  },

  scroll: {
    '& div': {
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: '#EEEEEE'
      }
    }
  },
  loadingText: {
    marginBottom: 0,
    paddingLeft: theme.spacing(0.5)
  },
  AIoutputFilter: {
    '& p': {
      color: theme.colors.grey[9],
      fontStyle: 'italic',
      fontWeight: 400,
      fontSize: '0.6875rem'
    },
    '& svg': {
      cursor: 'pointer'
    }
  },
  lightAIoutputFilter: {
    '& p': {
      color: theme.colors.grey[10],
      fontStyle: 'italic',
      fontWeight: 400,
      fontSize: '0.6875rem'
    },
    '& svg': {
      cursor: 'pointer'
    }
  },
  popperFilter: {
    display: 'flex',
    alignItems: 'center',
    padding: theme.spacing(1.5, 1),
    width: '227px !important',
    backgroundColor: theme.colors.grey[16],
    border: `0.4px solid ${theme.colors.grey[12]}`,
    boxShadow: theme.colors.shadow.xl,
    bordeRadius: '6px',
    cursor: 'default'
  },
  lightPopperFilter: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    padding: theme.spacing(1.5, 1.5, 1.5, 1),
    width: '227px !important',
    backgroundColor: theme.colors.grey[0],
    border: `0.4px solid ${theme.colors.grey[2]}`,
    boxShadow: theme.colors.shadow.xl,
    bordeRadius: '6px',
    cursor: 'default'
  },
  clearFilterBtn: {
    border: 'none'
  }
}));

export default useStyles;
