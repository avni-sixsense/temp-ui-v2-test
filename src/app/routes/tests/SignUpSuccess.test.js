// This would create a snapshot of this page

import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
import theme from 'app/configs/theme';
import SignupSuccess from 'app/main/apps/signup/SignupSuccess';

describe('<LoadingPage /> should render', () => {
  it('should work', () => {
    const wrapper = renderer
      .create(
        <MuiThemeProvider theme={theme}>
          <SignupSuccess />
        </MuiThemeProvider>
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
