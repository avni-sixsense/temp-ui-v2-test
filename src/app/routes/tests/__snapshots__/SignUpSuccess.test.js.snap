// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingPage /> should render should work 1`] = `
<div
  className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation0 MuiPaper-rounded"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item"
    >
      <img
        alt=""
        className="makeStyles-successImg-4"
        src="group-29.svg"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
    >
      <h1
        className="MuiTypography-root makeStyles-title-2 MuiTypography-h1"
      >
        Signup Successful
      </h1>
      <h1
        className="MuiTypography-root makeStyles-subTitle-3 MuiTypography-h1"
      >
        We are processing your request from our end. Your admin will be sending out the credentials over the email.
      </h1>
    </div>
  </div>
</div>
`;
